Django_Blog/blog/models.py

```diff
@@ -1,3 +1,29 @@
 from django.db import models
 
 # Create your models here.
+class Author(models.Model):
+    name = models.CharField(max_length=100,null=False)
+    about_author = models.TextField()
+
+    def __str__(self):
+        return self.name
+
+class Blog(models.Model):
+    author = models.ForeignKey(Author, on_delete=models.CASCADE)
```

Comment:
Als add the `related_name` field to the foreignkey. Remember to add the `related_name` in other foreignkeys or the new foreignkeys you may create in the future.